src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color,double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: "
```

Comment:
Consider using a constant for the figure type ('square') instead of hardcoding it in the 'draw' method. This will make the code more maintainable and less error-prone if the figure type needs to be changed or referenced elsewhere.